packages/babel-core/test/__snapshots__/option-manager.js.snap

```diff
@@ -3,32 +3,32 @@
 exports[`option-manager config plugin/preset flattening and overriding should throw when an option is following a preset 1`] = `
 "[BABEL] unknown: Unknown option: .useSpread. Check out https://babeljs.io/docs/en/babel-core/#options for more information about options.
 - Maybe you meant to use
-\\"preset\\": [
+\\"presets\\": [
   [\\"./fixtures/option-manager/babel-preset-bar\\", {
   \\"useSpread\\": true
 }]
 ]
-To be a valid preset, its name and options should be wrapped in a pair of brackets"
+To be a valid presets, its name and options should be wrapped in a pair of brackets"
```

Comment:
Actually, it's better not to have the `s` here. Maybe you could edit the `checkNoUnwrappedItemOptionPairs` adding the `s` for the "Maybe you meant to use" section, rather than changing it in the parameter?

Reply:
good point!
I fixed it as you said.

b8d5987